script_library/deployment-actions-kvm.sh

```diff
@@ -56,7 +56,8 @@ function deploy_airship(){
 }
 function add_compute(){
     echo "Now Add compute"
-    run_ansible ${socok8s_absolute_dir}/playbooks/add_compute.yml
+    tagged_info=" --tags add_compute_node"
```

Comment:
Not sure what's the benefit of having a variable and using it in next line as-is